src/goto-programs/generate_function_bodies.cpp

```diff
@@ -8,6 +8,11 @@ Author: Diffblue Ltd.
 
 #include "generate_function_bodies.h"
 
+#include <ansi-c/c_nondet_symbol_factory.h>
+#include <ansi-c/c_object_factory_parameters.h>
```

Comment:
We should not have dependencies on `ansi-c` in `goto-programs`. Some re-architecting is needed here.

Reply:
The last commit actually moves the function body generation from `goto-programs` to `goto-instrument`.

Reply:
Uh, ok, can you re-arrange things or squash commits to never make the above change appear?